user/serializers.py

```diff
@@ -1 +1,23 @@
-# write your code here
+from django.contrib.auth import get_user_model
+from rest_framework import serializers
+
+
+class UserSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = get_user_model()
+        fields = ("id", "username", "email", "password", "is_staff")
+        read_only_fields = ("id", "is_staff")
+        extra_kwargs = {
+            "password": {"write_only": True, "min_length": 5}
+        }
+
+    def create(self, validated_data):
+        return get_user_model().objects.create_user(**validated_data)
```

Comment:
When creating a user, ensure that the password is hashed. The `create_user` method typically handles this, but it's important to verify that `create_user` is implemented correctly in your custom user model.